packages/lib/src/vis/shared/SelectionTool.tsx

```diff
@@ -77,24 +82,49 @@ function SelectionTool(props: Props) {
       const { sourceEvent, unprojectedPoint } = evt;
       const { target, pointerId } = sourceEvent;
       (target as Element).releasePointerCapture(pointerId);
+      setStartPoint(undefined);
+      setEndPoint(undefined);
```

Comment:
Releasing the mouse should reset the selection whether the modifier key is pressed or not